Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -63,18 +63,32 @@ + (NSDictionary *)appLaunchEnvironmentWithBundleID:(NSString *)hostBundleID
+                                                                  @"DYLD_FALLBACK_LIBRARY_PATH" : [NSString stringWithFormat:@"%@/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib", config.xcodePath],
                                                                   @"DYLD_INSERT_LIBRARIES" : [NSString stringWithFormat:@"%@/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/Developer/usr/lib/libXCTTargetBootstrapInject.dylib", config.xcodePath],
-                                                                  @"DYLD_LIBRARY_PATH" : dyldLibraryPath,
+                                                                  @"DYLD_LIBRARY_PATH" : [NSString stringWithFormat:@"%@/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks", config.xcodePath],
+                                                                  @"DYLD_ROOT_PATH" : [NSString stringWithFormat:@"%@/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot", config.xcodePath],
                                                                   @"NSUnbufferedIO" : @YES,
                                                                   @"OS_ACTIVITY_DT_MODE" : @YES,
-                                                                  @"MNTF_TINKER_DELAY": @0.01,
+                                                                  @"NSUnbufferedIO" : @YES,
+                                                                  @"MNTF_SCREENSHOTS" : @"$(PROJECT_DIR)/build/outputs/tests_artifacts",
+                                                                  @"MNTF_SCREENSHOTS_BASELINE" : @"$(PROJECT_DIR)/mntf-iosUITests/Screenshots/Baseline",
+                                                                  @"MNTF_SCREENSHOTS_BASELINE_REAL" : @"/Documents/Baseline",
+                                                                  @"MNTF_SCREENSHOTS_DIFF" : @"$(PROJECT_DIR)/build/outputs/tests_artifacts/Diff",
+                                                                  @"MNTF_SCREENSHOTS_DIFF_REAL" : @"/Documents/Screenshots/Diff",
+                                                                  @"MNTF_SCREENSHOTS_REAL" : @"/Documents/Screenshots",
+                                                                  @"MNTF_TINKER_DELAY" : @0.01,
+                                                                  @"SIMULATOR_HID_SYSTEM_MANAGER" : @"/Library/Developer/PrivateFrameworks/CoreSimulator.framework/Resources/Platforms/iphoneos/Library/Frameworks/SimulatorHID.framework",
+                                                                  @"SIMULATOR_MODEL_IDENTIFIER" : @"iPhone9,1",
+                                                                  @"SIMULATOR_RUNTIME_VERSION" : @12.0,
```

Comment:
I'm also not sure we should be hard-coding these... what if the config file specifies a different simulator or runtime?

Reply:
yeah, they should not be set here.